EosioSwift/EosioRpcProvider/Models/RpcProviderResponseModels.swift

```diff
@@ -291,22 +291,26 @@ public struct Authority: Decodable {
     public var threshold: EosioUInt64
     public var keys: [KeyWeight]
     public var waits: [WaitWeight]
+    public var accounts: [EosioName]
```

Comment:
I don't think accounts is supposed to be an `[EosioName]`. According to https://github.com/EOSIO/swagger/blob/master/swagger/Schemas/RequiredAuthority.yaml, I see this structure:

```js
{
    waits: [WaitWeight],
    keys: [KeyWeight],
    threshold: EosioUInt64,
    accounts: [{
        weight: EosioUInt64,
        permission: {
            actor: EosioName,
            permission: EosioName
        }
    }]
```

I didn't find any structs with `weight` and `permission`. I'm thinking it's new for us. Might even be two new structs.

Reply:
I agree that looks like the correct account structure from what I can see in other EOSIO repos.

WaitWeight is at: https://github.com/EOSIO/eosio-swift/blob/318269ed2684c1c90e1db7436e441a83f8a420c3/EosioSwift/EosioRpcProvider/Models/RpcProviderResponseModels.swift#L246

KeyWeight is at: https://github.com/EOSIO/eosio-swift/blob/318269ed2684c1c90e1db7436e441a83f8a420c3/EosioSwift/EosioRpcProvider/Models/RpcProviderResponseModels.swift#L279

So I think those are ok.

Reply:
PermissionLevelWeight and PermissionLevel match what is under accounts.

Reply:
> PermissionLevelWeight and PermissionLevel match what is under accounts.

I didn't think these matched up properly. Seems the value of accounts might be made up of two structs we do not yet have.

Reply:
weight in account is just a field, not a struct from how I'm reading it?

Reply:
```
/// Response type for `permission_level` in RPC endpoint responses.
public struct PermissionLevel: Decodable {
    public var actor: String
    public var permission: String

    enum CodingKeys: String, CodingKey {
        case actor
        case permission
    }
}

/// Response type for `permission_level_weight in RPC endpoint responses.
public struct PermissionLevelWeight: Decodable {
    public var weight: EosioUInt64
    public var accounts: [PermissionLevel]

    enum CodingKeys: String, CodingKey {
        case weight
        case accounts
    }
}
```
The field naming is off but the content is the same at both levels.

Reply:
Guys, you have very good eyes 👀
I have updated my object to match the structure in Brandon's comment.